brili.ts

```diff
@@ -330,8 +335,12 @@ type Action =
   | { "action": "end"; "ret": Value | null }
   | { "action": "speculate" }
   | { "action": "commit" }
-  | { "action": "abort"; "label": bril.Ident };
+  | { "action": "abort", "label": bril.Ident }
+  | { "action": "break", "level": number } // Break out of n loops.
+  | { "action": "continue", "level": number }; // Continue to next iteration of innermost loop.
```

Comment:
The comment appears out-of-date, because `continue` also has a level parameter (so it's not just the innermost loop).